### ISSUE TITLE
Label wrap are wrong sometimes

### ISSUE THREAD

**Issue by tito** [kivy/kivy#2503]
This wasn't an issue in 1.8 before, but it is now:

```python
from kivy.base import runTouchApp
from kivy.uix.label import Label
runTouchApp(Label(
    text="This is a big line of text hell yeah.",
    text_size=(100, None)))
```

Result:
![image](https://cloud.githubusercontent.com/assets/37904/4371418/41d063f4-4317-11e4-945d-309d92adc644.png)

**Comment by matham** [comment]
Reopened to be fixed by #2840.

**Comment by akshayaurora** [comment]
closed by #2840
